slip.js

```diff
@@ -454,21 +454,25 @@ window['Slip'] = (function(){
 
                     onEnd: function() {
                         var move = this.getTotalMovement();
+                        var i, spliceIndex;
                         if (move.y < 0) {
-                            for(var i=0; i < otherNodes.length; i++) {
+                            for (i=0; i < otherNodes.length; i++) {
                                 if (otherNodes[i].pos > move.y) {
-                                    this.dispatch(this.target.node, 'reorder', {spliceIndex:i, insertBefore:otherNodes[i].node, originalIndex: originalIndex});
```

Comment:
The old code would dispatch the event in this for-loop, but what if the condition is never met?